aptos-move/framework/aptos-framework/sources/vesting_without_staking.move

```diff
@@ -0,0 +1,1174 @@
+            },
+        );
+    }
+
+    /// Remove the beneficiary for the given shareholder. All distributions will sent directly to the shareholder
+    /// account.
+    public entry fun reset_beneficiary(
+        account: &signer,
+        contract_address: address,
+        shareholder: address,
+    ) acquires VestingAccountManagement, VestingContract {
+        let vesting_contract = borrow_global_mut<VestingContract>(contract_address);
+        let addr = signer::address_of(account);
+        assert!(
+            addr == vesting_contract.admin ||
+                addr == get_role_holder(contract_address, utf8(ROLE_BENEFICIARY_RESETTER)),
+            error::permission_denied(EPERMISSION_DENIED),
+        );
+
+        let beneficiaries = &mut vesting_contract.beneficiaries;
+        if (simple_map::contains_key(beneficiaries, &shareholder)) {
+            simple_map::remove(beneficiaries, &shareholder);
+        };
+    }
+
+    public entry fun set_management_role(
+        admin: &signer,
+        contract_address: address,
+        role: String,
+        role_holder: address,
+    ) acquires VestingAccountManagement, VestingContract {
+        let vesting_contract = borrow_global_mut<VestingContract>(contract_address);
+        verify_admin(admin, vesting_contract);
+
+        if (!exists<VestingAccountManagement>(contract_address)) {
+            let contract_signer = &get_vesting_account_signer_internal(vesting_contract);
+            move_to(contract_signer, VestingAccountManagement {
+                roles: simple_map::create<String, address>(),
+            })
+        };
+        let roles = &mut borrow_global_mut<VestingAccountManagement>(contract_address).roles;
+        if (simple_map::contains_key(roles, &role)) {
+            *simple_map::borrow_mut(roles, &role) = role_holder;
+        } else {
+            simple_map::add(roles, role, role_holder);
+        };
+    }
+
+    public entry fun set_beneficiary_resetter(
+        admin: &signer,
+        contract_address: address,
+        beneficiary_resetter: address,
+    ) acquires VestingAccountManagement, VestingContract {
+        set_management_role(admin, contract_address, utf8(ROLE_BENEFICIARY_RESETTER), beneficiary_resetter);
+    }
+
+    public fun get_role_holder(contract_address: address, role: String): address acquires VestingAccountManagement {
+        assert!(exists<VestingAccountManagement>(contract_address), error::not_found(EVESTING_ACCOUNT_HAS_NO_ROLES));
+        let roles = &borrow_global<VestingAccountManagement>(contract_address).roles;
+        assert!(simple_map::contains_key(roles, &role), error::not_found(EROLE_NOT_FOUND));
+        *simple_map::borrow(roles, &role)
+    }
+
+    /// For emergency use in case the admin needs emergency control of vesting contract account.
+    /// This doesn't give the admin total power as the admin would still need to follow the rules set by
+    /// staking_contract and stake modules.
```

Comment:
Update the comments to reflect everywhere that this does not have `staking`.

Reply:
Updated